src/logic/catalog/queries.ts

```diff
@@ -0,0 +1,503 @@
+import { IPgComponent } from '@well-known-components/pg-component'
```

Comment:
I was waiting to incorporate the Favorites component to move the test since they use the favorites picks in several places. WDYT? I can put them in the next PR alongside the favorites component.